packages/vant/package.json

```diff
@@ -48,13 +48,15 @@
   "dependencies": {
     "@vant/popperjs": "workspace:^",
     "@vant/use": "workspace:^",
-    "@vue/shared": "^3.0.0"
+    "@vue/shared": "^3.0.0",
+    "transliteration": "^2.3.5"
```

Comment:
we want no third-party dependencies.

Reply:
yes, maybe we do not need the `slugifyTitle` in this component